stream-chat-android-client/src/main/java/io/getstream/chat/android/client/models/Config.kt

```diff
@@ -109,4 +116,25 @@ public data class Config(
      * The commands that are available on this channel type, e.g. /giphy.
      */
     val commands: List<Command> = mutableListOf(),
-)
+) {
+    val createdAt: Date?
```

Comment:
We should add docs on these properties, just like the original ones had kdoc comments